---
title: "RNA Processing in Eukaryotes"
layout: page
---


<div data-type="abstract" markdown="1">
By the end of this section, you will be able to:
* Describe the different steps in RNA processing
* Understand the significance of exons, introns, and splicing
* Explain how tRNAs and rRNAs are processed

</div>

After transcription, eukaryotic pre-mRNAs must undergo several processing steps before they can be translated. Eukaryotic (and prokaryotic) tRNAs and rRNAs also undergo processing before they can function as components in the protein synthesis machinery.

### mRNA Processing

The eukaryotic pre-mRNA undergoes extensive processing before it is ready to be translated. The additional steps involved in eukaryotic mRNA maturation create a molecule with a much longer half-life than a prokaryotic mRNA. Eukaryotic mRNAs last for several hours, whereas the typical *E. coli* mRNA lasts no more than five seconds.

Pre-mRNAs are first coated in RNA-stabilizing proteins; these protect the pre-mRNA from degradation while it is processed and exported out of the nucleus. The three most important steps of pre-mRNA processing are the addition of stabilizing and signaling factors at the 5\' and 3\' ends of the molecule, and the removal of intervening sequences that do not specify the appropriate amino acids. In rare cases, the mRNA transcript can be “edited” after it is transcribed.

<div data-type="note" data-has-label="true" class="evolution" data-label="" markdown="1">
<div data-type="title">
Evolution Connection
</div>
<span data-type="title">RNA Editing in Trypanosomes</span>The trypanosomes are a group of protozoa that include the pathogen *Trypanosoma brucei*, which causes sleeping sickness in humans ([\[link\]](#fig-ch15_04_01)). Trypanosomes, and virtually all other eukaryotes, have organelles called mitochondria that supply the cell with chemical energy. Mitochondria are organelles that express their own DNA and are believed to be the remnants of a symbiotic relationship between a eukaryote and an engulfed prokaryote. The mitochondrial DNA of trypanosomes exhibit an interesting exception to The Central Dogma: their pre-mRNAs do not have the correct information to specify a functional protein. Usually, this is because the mRNA is missing several U nucleotides. The cell performs an additional RNA processing step called **RNA editing**{: data-type="term"} to remedy this.

![Micrograph shows T. brucei, which has a u-shaped cell body and a long tail.](../resources/Figure_15_04_01.jpg "Trypanosoma brucei is the causative agent of sleeping sickness in humans. The mRNAs of this pathogen must be modified by the addition of nucleotides before protein synthesis can occur. (credit: modification of work by Torsten Ochsenreiter)"){: #fig-ch15_04_01}


Other genes in the mitochondrial genome encode 40- to 80-nucleotide guide RNAs. One or more of these molecules interacts by complementary base pairing with some of the nucleotides in the pre-mRNA transcript. However, the guide RNA has more A nucleotides than the pre-mRNA has U nucleotides to bind with. In these regions, the guide RNA loops out. The 3\' ends of guide RNAs have a long poly-U tail, and these U bases are inserted in regions of the pre-mRNA transcript at which the guide RNAs are looped. This process is entirely mediated by RNA molecules. That is, guide RNAs—rather than proteins—serve as the catalysts in RNA editing.

RNA editing is not just a phenomenon of trypanosomes. In the mitochondria of some plants, almost all pre-mRNAs are edited. RNA editing has also been identified in mammals such as rats, rabbits, and even humans. What could be the evolutionary reason for this additional step in pre-mRNA processing? One possibility is that the mitochondria, being remnants of ancient prokaryotes, have an equally ancient RNA-based method for regulating gene expression. In support of this hypothesis, edits made to pre-mRNAs differ depending on cellular conditions. Although speculative, the process of RNA editing may be a holdover from a primordial time when RNA molecules, instead of proteins, were responsible for catalyzing reactions.

</div>

#### 5\' Capping

While the pre-mRNA is still being synthesized, a **7-methylguanosine cap**{: data-type="term"} is added to the 5\' end of the growing transcript by a phosphate linkage. This moiety (functional group) protects the nascent mRNA from degradation. In addition, factors involved in protein synthesis recognize the cap to help initiate translation by ribosomes.

#### 3\' Poly-A Tail

Once elongation is complete, the pre-mRNA is cleaved by an endonuclease between an AAUAAA consensus sequence and a GU-rich sequence, leaving the AAUAAA sequence on the pre-mRNA. An enzyme called poly-A polymerase then adds a string of approximately 200 A residues, called the **poly-A tail**{: data-type="term"}. This modification further protects the pre-mRNA from degradation and signals the export of the cellular factors that the transcript needs to the cytoplasm.

#### Pre-mRNA Splicing

Eukaryotic genes are composed of **exons**{: data-type="term"}, which correspond to protein-coding sequences (*ex-*on signifies that they are *ex*pressed), and *int*ervening sequences called **introns**{: data-type="term"} (*int-*ron denotes their *int*ervening role), which may be involved in gene regulation but are removed from the pre-mRNA during processing. Intron sequences in mRNA do not encode functional proteins.

The discovery of introns came as a surprise to researchers in the 1970s who expected that pre-mRNAs would specify protein sequences without further processing, as they had observed in prokaryotes. The genes of higher eukaryotes very often contain one or more introns. These regions may correspond to regulatory sequences; however, the biological significance of having many introns or having very long introns in a gene is unclear. It is possible that introns slow down gene expression because it takes longer to transcribe pre-mRNAs with lots of introns. Alternatively, introns may be nonfunctional sequence remnants left over from the fusion of ancient genes throughout evolution. This is supported by the fact that separate exons often encode separate protein subunits or domains. For the most part, the sequences of introns can be mutated without ultimately affecting the protein product.

All of a pre-mRNA’s introns must be completely and precisely removed before protein synthesis. If the process errs by even a single nucleotide, the reading frame of the rejoined exons would shift, and the resulting protein would be dysfunctional. The process of removing introns and reconnecting exons is called **splicing**{: data-type="term"} ([\[link\]](#fig-ch15_04_02)). Introns are removed and degraded while the pre-mRNA is still in the nucleus. Splicing occurs by a sequence-specific mechanism that ensures introns will be removed and exons rejoined with the accuracy and precision of a single nucleotide. The splicing of pre-mRNAs is conducted by complexes of proteins and RNA molecules called spliceosomes.

<div data-type="note" data-has-label="true" class="art-connection" data-label="" markdown="1">
<div data-type="title">
Art Connection
</div>
![Illustration shows a spliceosome bound to mRNA. An intron is wrapped around snRNPs associated with the spliceosome. When the splice is complete, the exons on either side of the intron are fused together, and the intron forms a ring structure.](../resources/Figure_15_04_02.png "Pre-mRNA splicing involves the precise removal of introns from the primary RNA transcript. The splicing process is catalyzed by protein complexes called spliceosomes that are composed of proteins and RNA molecules called snRNAs. Spliceosomes recognize sequences at the 5' and 3' end of the intron."){: #fig-ch15_04_02}


Errors in splicing are implicated in cancers and other human diseases. What kinds of mutations might lead to splicing errors? Think of different possible outcomes if splicing errors occur.

<!-- <link window="new"  target-id="fig-ch15_04_02" document=""/>Mutations in the spliceosome recognition sequence at each end of the intron, or in the proteins and RNAs that make up the spliceosome, may impair splicing. Mutations may also add new spliceosome recognition sites. Splicing errors could lead to introns being retained in spliced RNA, exons being excised, or changes in the location of the splice site. -->

</div>

Note that more than 70 individual introns can be present, and each has to undergo the process of splicing—in addition to 5\' capping and the addition of a poly-A tail—just to generate a single, translatable mRNA molecule.

<div data-type="note" data-has-label="true" class="interactive" data-label="" markdown="1">
<div data-type="title">
Link to Learning
</div>
<span data-type="media" data-alt="QR Code representing a URL"> ![QR Code representing a URL](../resources/RNA_splicing.png) </span>
See how introns are removed during RNA splicing [at this website][1]{: target="_window"}.

</div>

### Processing of tRNAs and rRNAs

The tRNAs and rRNAs are structural molecules that have roles in protein synthesis; however, these RNAs are not themselves translated. Pre-rRNAs are transcribed, processed, and assembled into ribosomes in the nucleolus. Pre-tRNAs are transcribed and processed in the nucleus and then released into the cytoplasm where they are linked to free amino acids for protein synthesis.

Most of the tRNAs and rRNAs in eukaryotes and prokaryotes are first transcribed as a long precursor molecule that spans multiple rRNAs or tRNAs. Enzymes then cleave the precursors into subunits corresponding to each structural RNA. Some of the bases of pre-rRNAs are methylated; that is, a –CH<sub>3</sub> moiety (methyl functional group) is added for stability. Pre-tRNA molecules also undergo methylation. As with pre-mRNAs, subunit excision occurs in eukaryotic pre-RNAs destined to become tRNAs or rRNAs.

Mature rRNAs make up approximately 50 percent of each ribosome. Some of a ribosome’s RNA molecules are purely structural, whereas others have catalytic or binding activities. Mature tRNAs take on a three-dimensional structure through intramolecular hydrogen bonding to position the amino acid binding site at one end and the **anticodon**{: data-type="term"} at the other end ([\[link\]](#fig-ch15_04_03)). The anticodon is a three-nucleotide sequence in a tRNA that interacts with an mRNA codon through complementary base pairing.

 ![The molecular model of phenylalanine tRNA is L-shaped. At one end is the anticodon AAG. At the other end is the attachment site for the amino acid phenylalanine](../resources/Figure_15_04_03.jpg "This is a space-filling model of a tRNA molecule that adds the amino acid phenylalanine to a growing polypeptide chain. The anticodon AAG binds the Codon UUC on the mRNA. The amino acid phenylalanine is attached to the other end of the tRNA."){: #fig-ch15_04_03}

### Section Summary

Eukaryotic pre-mRNAs are modified with a 5\' methylguanosine cap and a poly-A tail. These structures protect the mature mRNA from degradation and help export it from the nucleus. Pre-mRNAs also undergo splicing, in which introns are removed and exons are reconnected with single-nucleotide accuracy. Only finished mRNAs that have undergone 5\' capping, 3\' polyadenylation, and intron splicing are exported from the nucleus to the cytoplasm. Pre-rRNAs and pre-tRNAs may be processed by intramolecular cleavage, splicing, methylation, and chemical conversion of nucleotides. Rarely, RNA editing is also performed to insert missing bases after an mRNA has been synthesized.

### Art Connections

<div data-type="exercise">
<div data-type="problem" markdown="1">
[[link]](#fig-ch15_04_02) Errors in splicing are implicated in cancers and other human diseases. What kinds of mutations might lead to splicing errors? Think of different possible outcomes if splicing errors occur.

</div>
<div data-type="solution" markdown="1">
[[link]](#fig-ch15_04_02) Mutations in the spliceosome recognition sequence at each end of the intron, or in the proteins and RNAs that make up the spliceosome, may impair splicing. Mutations may also add new spliceosome recognition sites. Splicing errors could lead to introns being retained in spliced RNA, exons being excised, or changes in the location of the splice site.

</div>
</div>

### Review Questions

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which pre-mRNA processing step is important for initiating translation?

1.  poly-A tail
2.  RNA editing
3.  splicing
4.  7-methylguanosine cap
{: type="a"}

</div>
<div data-type="solution" markdown="1">
D

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
What processing step enhances the stability of pre-tRNAs and pre-rRNAs?

1.  methylation
2.  nucleotide modification
3.  cleavage
4.  splicing
{: type="a"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

<div data-type="glossary" markdown="1">
### Glossary
{: data-type="glossary-title"}

7-methylguanosine cap
: modification added to the 5\' end of pre-mRNAs to protect mRNA from degradation and assist translation
^

anticodon
: three-nucleotide sequence in a tRNA molecule that corresponds to an mRNA codon
^

exon
: sequence present in protein-coding mRNA after completion of pre-mRNA splicing
^

intron
: non–protein-coding intervening sequences that are spliced from mRNA during processing
^

poly-A tail
: modification added to the 3\' end of pre-mRNAs to protect mRNA from degradation and assist mRNA export from the nucleus
^

RNA editing
: direct alteration of one or more nucleotides in an mRNA that has already been synthesized
^

splicing
: process of removing introns and reconnecting exons in a pre-mRNA

</div>



[1]: http://openstaxcollege.org/l/RNA_splicing
