---
title: "Eukaryotic Epigenetic Gene Regulation"
layout: page
---


<div data-type="abstract" markdown="1">
By the end of this section, you will be able to:
* Explain the process of epigenetic regulation
* Describe how access to DNA is controlled by histone modification

</div>

Eukaryotic gene expression is more complex than prokaryotic gene expression because the processes of transcription and translation are physically separated. Unlike prokaryotic cells, eukaryotic cells can regulate gene expression at many different levels. Eukaryotic gene expression begins with control of access to the DNA. This form of regulation, called epigenetic regulation, occurs even before transcription is initiated.

### Epigenetic Control: Regulating Access to Genes within the Chromosome

The human genome encodes over 20,000 genes; each of the 23 pairs of human chromosomes encodes thousands of genes. The DNA in the nucleus is precisely wound, folded, and compacted into chromosomes so that it will fit into the nucleus. It is also organized so that specific segments can be accessed as needed by a specific cell type.

The first level of organization, or packing, is the winding of DNA strands around histone proteins. Histones package and order DNA into structural units called nucleosome complexes, which can control the access of proteins to the DNA regions ([\[link\]](#fig-ch16_03_01)**a**). Under the electron microscope, this winding of DNA around histone proteins to form nucleosomes looks like small beads on a string ([\[link\]](#fig-ch16_03_01)**b**). These beads (histone proteins) can move along the string (DNA) and change the structure of the molecule.

![Part A depicts a nucleosome composed of spherical histone proteins that are fused together. A double-stranded DNA helix wraps around the nucleosome twice. Free DNA extends from either end of the nucleosome.  Part B is an electron micrograph of DNA that is associated with nucleosomes. Each nucleosome looks like a bead. The beads are connected together by free DNA. Nine beads strung together is approximately 150 nm across.](../resources/Figure_16_03_01ab.jpg "DNA is folded around histone proteins to create (a) nucleosome complexes. These nucleosomes control the access of proteins to the underlying DNA. When viewed through an electron microscope (b), the nucleosomes look like beads on a string. (credit &#x201C;micrograph&#x201D;: modification of work by Chris Woodcock)"){: #fig-ch16_03_01}

If DNA encoding a specific gene is to be transcribed into RNA, the nucleosomes surrounding that region of DNA can slide down the DNA to open that specific chromosomal region and allow for the transcriptional machinery (RNA polymerase) to initiate transcription ([\[link\]](#fig-ch16_03_02)). Nucleosomes can move to open the chromosome structure to expose a segment of DNA, but do so in a very controlled manner.

<div data-type="note" data-has-label="true" class="art-connection" data-label="" markdown="1">
<div data-type="title">
Art Connection
</div>
![Nucleosomes are depicted as wheel-like structures. The nucleosomes are made up of histones, and have DNA wrapped around the outside. Each histone has a tail that juts out from the wheel. When DNA and the histone tails are methylated, the nucleosomes pack tightly together so there is no free DNA. Transcription factors cannot bind, and genes are not expressed. Acetylation of histone tails results in a looser packing of the nucleosomes. Free DNA is exposed between the nucleosomes, and transcription factors are able to bind genes on this exposed DNA.](../resources/Figure_16_03_02.png "Nucleosomes can slide along DNA. When nucleosomes are spaced closely together (top), transcription factors cannot bind and gene expression is turned off. When the nucleosomes are spaced far apart (bottom), the DNA is exposed. Transcription factors can bind, allowing gene expression to occur. Modifications to the histones and DNA affect nucleosome spacing."){: #fig-ch16_03_02}


In females, one of the two X chromosomes is inactivated during embryonic development because of epigenetic changes to the chromatin. What impact do you think these changes would have on nucleosome packing?

<!-- <link window="new"  target-id="fig-ch16_03_02" document=""/> The nucleosomes would pack more tightly together. -->

</div>

How the histone proteins move is dependent on signals found on both the histone proteins and on the DNA. These signals are tags added to histone proteins and DNA that tell the histones if a chromosomal region should be open or closed ([\[link\]](#fig-ch16_03_03) depicts modifications to histone proteins and DNA). These tags are not permanent, but may be added or removed as needed. They are chemical modifications (phosphate, methyl, or acetyl groups) that are attached to specific amino acids in the protein or to the nucleotides of the DNA. The tags do not alter the DNA base sequence, but they do alter how tightly wound the DNA is around the histone proteins. DNA is a negatively charged molecule; therefore, changes in the charge of the histone will change how tightly wound the DNA molecule will be. When unmodified, the histone proteins have a large positive charge; by adding chemical modifications like acetyl groups, the charge becomes less positive.

The DNA molecule itself can also be modified. This occurs within very specific regions called CpG islands. These are stretches with a high frequency of cytosine and guanine dinucleotide DNA pairs (CG) found in the promoter regions of genes. When this configuration exists, the cytosine member of the pair can be methylated (a methyl group is added). This modification changes how the DNA interacts with proteins, including the histone proteins that control access to the region. Highly methylated (hypermethylated) DNA regions with deacetylated histones are tightly coiled and transcriptionally inactive.

 ![Illustration shows a chromosome that is partially unraveled and magnified, revealing histone proteins wound around the DNA double helix. Histones are proteins around which DNA winds for compaction and gene regulation. Methylation of DNA and chemical modification of histone tails are known as epigenetic changes. Epigenetic changes alter the spacing of nucleosomes and change gene expression. Epigenetic changes may result from development, either in utero or in childhood, environmental chemicals, drugs, aging, or diet. Epigenetic changes may result in cancer, autoimmune disease, mental disorders, and diabetes.](../resources/Figure_16_03_03.jpg "Histone proteins and DNA nucleotides can be modified chemically. Modifications affect nucleosome spacing and gene expression. (credit: modification of work by NIH)"){: #fig-ch16_03_03}

This type of gene regulation is called epigenetic regulation. Epigenetic means “around genetics.” The changes that occur to the histone proteins and DNA do not alter the nucleotide sequence and are not permanent. Instead, these changes are temporary (although they often persist through multiple rounds of cell division) and alter the chromosomal structure (open or closed) as needed. A gene can be turned on or off depending upon the location and modifications to the histone proteins and DNA. If a gene is to be transcribed, the histone proteins and DNA are modified surrounding the chromosomal region encoding that gene. This opens the chromosomal region to allow access for RNA polymerase and other proteins, called **transcription factors**{: data-type="term"}, to bind to the promoter region, located just upstream of the gene, and initiate transcription. If a gene is to remain turned off, or silenced, the histone proteins and DNA have different modifications that signal a closed chromosomal configuration. In this closed configuration, the RNA polymerase and transcription factors do not have access to the DNA and transcription cannot occur ([\[link\]](#fig-ch16_03_02)).

<div data-type="note" data-has-label="true" class="interactive-embedded-reading" data-label="" markdown="1">
<div data-type="title">
Link to Learning
</div>
<span data-type="media" data-alt="QR Code representing a URL"> ![QR Code representing a URL](../resources/epigenetic_reg.png) </span>
View this video that describes how epigenetic regulation controls gene expression.

<div data-type="media" id="eip-id1169842033590" data-alt="epigenetic_reg">
<iframe width="660" height="371.4" src="https://www.openstaxcollege.org/l/epigenetic_reg"></iframe>
</div>
</div>

### Section Summary

In eukaryotic cells, the first stage of gene expression control occurs at the epigenetic level. Epigenetic mechanisms control access to the chromosomal region to allow genes to be turned on or off. These mechanisms control how DNA is packed into the nucleus by regulating how tightly the DNA is wound around histone proteins. The addition or removal of chemical modifications (or flags) to histone proteins or DNA signals to the cell to open or close a chromosomal region. Therefore, eukaryotic cells can control whether a gene is expressed by controlling accessibility to transcription factors and the binding of RNA polymerase to initiate transcription.

### Art Connections

<div data-type="exercise">
<div data-type="problem" markdown="1">
[[link]](#fig-ch16_03_02) In females, one of the two X chromosomes is inactivated during embryonic development because of epigenetic changes to the chromatin. What impact do you think these changes would have on nucleosome packing?

</div>
<div data-type="solution" markdown="1">
[[link]](#fig-ch16_03_02) The nucleosomes would pack more tightly together.

</div>
</div>

### Review Questions

<div data-type="exercise">
<div data-type="problem" markdown="1">
What are epigenetic modifications?

1.  the addition of reversible changes to histone proteins and DNA
2.  the removal of nucleosomes from the DNA
3.  the addition of more nucleosomes to the DNA
4.  mutation of the DNA sequence
{: type="a"}

</div>
<div data-type="solution" markdown="1">
A

</div>
</div>

<div data-type="exercise">
<div data-type="problem" markdown="1">
Which of the following are true of epigenetic changes?

1.  allow DNA to be transcribed
2.  move histones to open or close a chromosomal region
3.  are temporary
4.  all of the above
{: type="a"}

</div>
<div data-type="solution" markdown="1">
D

</div>
</div>

### Free Response

<div data-type="exercise">
<div data-type="problem" markdown="1">
In cancer cells, alteration to epigenetic modifications turns off genes that are normally expressed. Hypothetically, how could you reverse this process to turn these genes back on?

</div>
<div data-type="solution" markdown="1">
You can create medications that reverse the epigenetic processes (to add histone acetylation marks or to remove DNA methylation) and create an open chromosomal configuration.

</div>
</div>

<div data-type="glossary" markdown="1">
### Glossary
{: data-type="glossary-title"}

transcription factor
: protein that binds to the DNA at the promoter or enhancer region and that influences transcription of a gene

</div>

